### PR TITLE
Remove infinite scroll.

### LINKED ISSUE #713: Infinite scroll on gallery is buggy. Arg.

### LINKED ISSUE #754: Infinite scroll
Now that we have an infinite scroll, on a search results page it is not accurate to say that the first 1-10 of [total search results] is being displayed.

Example: http://americanarchive.org/catalog?f%5Baccess_types%5D%5B%5D=online&q=tree&utf8=%E2%9C%93&view=list

Here it says we are showing 1-10 of 20, but with the infinite scroll, all 20 are appearing on this page, not 1-10.

### LINKED ISSUE #758: Remove Infinite Scroll
Related to #754, we have decided that we preferred the previous functionality better than the infinite scroll. Please remove the infinite scroll functionality and replace with previous functionality that required user to go to next page to view more search results.

### LINKED ISSUE #775: Infinite scroll is repeating search results
**To reproduce:** conduct a search that results in only a few results. The same results are getting added to the results page.

**Done when:** result set is not repeated due to infinite scroll script.

### LINKED ISSUE #776: Infinite scroll is adding document classes of the wrong style
**NOTE:** just discovered this on a branch I'm on. First verify that it's a problem in `master` before continuing. Also, this may be precluded by #755, so maybe work on that one first.

**To reproduce:**
  1. conduct a search that results in only a single item
  1. scroll down to trigger infinite scroll
  1. switch to grid view if you are in list view, or switch to list view if you are in grid view
  1. after page reloads, scroll down to trigger infinite scroll again.
  1. If you are now in grid view, then the items being added will have the list view style. But if you are now in list view, the items being added will have the grid view style.

### PR DESCRIPTION
**Pull request by mccalluc**
Fix #776, Fix #775, Fix #758, Fix #754, Fix #713 
@afred: review?

### CODE REVIEW

File: app/views/catalog/_document_gallery.erb [hunk 1]
```diff
@@ -1,28 +0,0 @@
...
-            $(response).find('#documents').html()
-          );
-        });
-      }
-    }
-  });
-</script>
-<div id="documents" class="row documents-grid-view">
-  <%= render_gallery_collection documents %>
-</div>
```
Comment (afred) [inline app/views/catalog/_document_gallery.erb]: @mccalluc we still want lines 26-27 don't we?

Comment (afred) [inline app/views/catalog/_document_gallery.erb]: ^ we need to override blacklight's default class declaration, replacing `class="row gallery"` with `class="row document-grid-view"`.

**Comment by mccalluc** [comment]
@afred: You're right. Added those lines back in.

**Comment by afred** [comment]
@mccalluc and sorry for my dupe tickets

**Pull request merged**